projects/robots/universal_robots/worlds/ure.wbt

```diff
@@ -338,7 +338,7 @@ ConveyorBelt {
   }
   borderHeight 0
   speed 0.2
-  textureAnimation 0.016
+  textureAnimation 1
```

Comment:
Note that `ConveyorBelt` PROTO does no longer have the `textureAnimation` field.
This world now throws many errors on load.